added several new samples and articles

romankhar opened the pull request:
also made some changes to the sub-titles of sections

romankhar committed:
added several new samples and articles

also made some changes to the sub-titles of sections

rabbah commented on README.md:
```diff
@@ -39,11 +39,24 @@
+
+*Tutorials and other material to help you learn OpenWhisk*
+
+- [Official OpenWhisk Tutorial](https://github.com/openwhisk/openwhisk-tutorial) - Interactive tutorials for learning how to use OpenWhisk.
```
Tutorial and later tutorials. Should both be plural?

romankhar replied:
fixed it (it was already there :-) )

rabbah commented on README.md:
```diff
@@ -39,11 +39,24 @@
 
+- [OpenWhisk 101 - Message Hub and Kafka Data Processing](https://github.com/IBM/openwhisk-data-processing-message-hub) - Example of using OpenWhisk with Message Hub and Kafka to consume and publish messages.
+- [Transit IoT](https://medium.com/openwhisk/transit-flexible-pipeline-for-iot-data-with-bluemix-and-openwhisk-4824cf20f1e0) - Comprehensive example of using OpenWhisk for IoT data processing that uses Docker, Node-RED, Message Hub (Kafka based), Object Storage, Spark and Bluemix Data Science Experience services for data analytics.
+- [Logistics Wizard](https://www.ibm.com/blogs/bluemix/2017/02/microservices-multi-compute-approach-using-cloud-foundry-openwhisk/) is an enterprise-grade sample application which leverages OpenWhisk and CloudFoundry to build 12-factor style application. It is a smart supply chain management solution that aims to simulate an environment running an ERP system.
```
application -> applications?

romankhar replied:
changed to "12-factor style applications"

rabbah commented on README.md:
```diff
@@ -39,11 +39,24 @@
+- [OpenWhisk 101 - Message Hub and Kafka Data Processing](https://github.com/IBM/openwhisk-data-processing-message-hub) - Example of using OpenWhisk with Message Hub and Kafka to consume and publish messages.
+- [Transit IoT](https://medium.com/openwhisk/transit-flexible-pipeline-for-iot-data-with-bluemix-and-openwhisk-4824cf20f1e0) - Comprehensive example of using OpenWhisk for IoT data processing that uses Docker, Node-RED, Message Hub (Kafka based), Object Storage, Spark and Bluemix Data Science Experience services for data analytics.
+- [Logistics Wizard](https://www.ibm.com/blogs/bluemix/2017/02/microservices-multi-compute-approach-using-cloud-foundry-openwhisk/) is an enterprise-grade sample application which leverages OpenWhisk and CloudFoundry to build 12-factor style application. It is a smart supply chain management solution that aims to simulate an environment running an ERP system.
+- [Web Actions](https://medium.com/openwhisk/web-actions-serverless-web-apps-with-openwhisk-f21db459f9ba) - This sample shows hot to use OpenWhisk to build a complete Web App.
```
hot -> how.

romankhar replied:
done

rabbah commented on README.md:
```diff
@@ -39,11 +39,24 @@
+- [Transit IoT](https://medium.com/openwhisk/transit-flexible-pipeline-for-iot-data-with-bluemix-and-openwhisk-4824cf20f1e0) - Comprehensive example of using OpenWhisk for IoT data processing that uses Docker, Node-RED, Message Hub (Kafka based), Object Storage, Spark and Bluemix Data Science Experience services for data analytics.
+- [Logistics Wizard](https://www.ibm.com/blogs/bluemix/2017/02/microservices-multi-compute-approach-using-cloud-foundry-openwhisk/) is an enterprise-grade sample application which leverages OpenWhisk and CloudFoundry to build 12-factor style application. It is a smart supply chain management solution that aims to simulate an environment running an ERP system.
+- [Web Actions](https://medium.com/openwhisk/web-actions-serverless-web-apps-with-openwhisk-f21db459f9ba) - This sample shows hot to use OpenWhisk to build a complete Web App.
+- [Dark vision](https://github.com/IBM-Bluemix/openwhisk-darkvisionapp) - application that shows how to use OpenWhisk, Cloudant, Watson Visual Recognition, Object Storage to process video frames, tag and recognize scenes.
```
capitalize "a" in "application".

rabbah replied:
same below for "d" in "demonstrates"

romankhar replied:
done

romankhar committed:
minor fixes per PR comments

romankhar commented:
all fixed and commited

jthomas reviewed (changes requested):
I want to keep the "applications" section with links to Github repositories. 
Can you move the links that point to Medium blog posts to the articles section?

jthomas commented:
@mrutkows can you give me write access to this repo? I've filed my CLA.

romankhar commented:
every one of those Medium articles has associated Repo with a complete sample - do we need to list it twice - once under Articles and once under Samples? I would rather have it under Samples since most people wont look at the list of articles when there are looking for a sample

csantanapr reviewed (changes requested):
I would like to see Sample Applications sections pointing to github repos. in addition a link to the companion blog/article should be added.
Each application bullet ideally would have two links the primary one to the repo, and secondary (optional) to the companion blog/article.

Also we should check if the repo itself should also have a link to the companion article.

In the Article section, should only be links to blog/article that don't have associated repo. and only a link to the article.

romankhar commented:
I replaced links under samples with links to GitHub and moved those without GitHub to "articles". Please take a look at this and feel free to edit directly - as soon as we are all done editing it, I will copy&paste into my fork and refresh the PR: https://docs.google.com/document/d/1PIYg1sUDV8wimnm5H9imVJ7ZPXEGLzD3LhW4qKzkQ_Y/edit?usp=sharing

romankhar committed:
made few updates to URLs per comments in PR

please edit the source document here if you would like to see any changes - you can make them directly: https://docs.google.com/document/d/1PIYg1sUDV8wimnm5H9imVJ7ZPXEGLzD3LhW4qKzkQ_Y/edit?usp=sharing

jthomas commented:
LGTM.
